t/lib/server.lua

```diff
@@ -28,6 +28,11 @@ function _M.hello1()
     ngx.say("hello1 world")
 end
 
+-- mock server for test the fix of overhead's value
+function _M.hello4()
```

Comment:
we can reuse `sleep1`: 

https://github.com/apache/incubator-apisix/blob/492880c05ee064ea7274b0b1cd52f00356eea04d/t/lib/server.lua#L68

you can take a look at: 
https://github.com/apache/incubator-apisix/blob/master/t/node/timeout-upstream.t#L49